admin/modules/style/themes.php

```diff
@@ -1094,9 +1094,16 @@ function checkAction(id)
 
 			foreach($colors as $color)
 			{
-				$color = explode("=", $color);
-
-				$properties['colors'][$color[0]] = $color[1];
+				$color = trim($color);
+				if(preg_match('/^[a-z0-9]+={1}[a-z0-9]+$/i', $color))
```

Comment:
The `{1}` here shouldn't be necessary. Also, this wouldn't allow uppercase letters, which I would expect it to allow. the pattern should probably be `a-zA-Z0-9`.

Reply:
{1} makes the message more user friendly. If you mistake in 3 naming at a stretch there will be exact same message thrice without indicating where is the problem.

a-zA-z not required. Thats silly, still massively used. There is case insensitive flag `/i` in place and thats all we need.

@euantorano